lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -3,6 +3,7 @@ import BitcoinClient from './BitcoinClient';
 import BitcoinTransactionModel from './models/BitcoinTransactionModel';
 import ErrorCode from './ErrorCode';
 import IBitcoinConfig from './IBitcoinConfig';
+import IBitcoinWallet from './interfaces/IBitcoinWallet';
```

Comment:
Would makes sense to export this interface explicitly in the exports in `index.ts, it's fine if you want to do it later.